UT4MasterServer.Services/Scoped/CleanupService.cs

```diff
@@ -0,0 +1,75 @@
+﻿using Microsoft.Extensions.Logging;
+using UT4MasterServer.Common;
+using UT4MasterServer.Services.Singleton;
+
+namespace UT4MasterServer.Services.Scoped;
+
+public sealed class CleanupService
+{
+	private readonly ILogger<CleanupService> logger;
+	private readonly AccountService accountService;
+	private readonly SessionService sessionService;
+	private readonly CodeService codeService;
+	private readonly FriendService friendService;
+	private readonly CloudStorageService cloudStorageService;
+	private readonly StatisticsService statisticsService;
+	private readonly TrustedGameServerService trustedGameServerService;
+	private readonly RatingsService ratingsService;
+
+	public CleanupService(
+		ILogger<CleanupService> logger,
+		AccountService accountService,
+		SessionService sessionService,
+		CodeService codeService,
+		FriendService friendService,
+		CloudStorageService cloudStorageService,
+		StatisticsService statisticsService,
+		TrustedGameServerService trustedGameServerService,
+		RatingsService ratingsService
+)
+	{
+		this.logger = logger;
+		this.accountService = accountService;
+		this.sessionService = sessionService;
+		this.codeService = codeService;
+		this.friendService = friendService;
+		this.cloudStorageService = cloudStorageService;
+		this.statisticsService = statisticsService;
+		this.trustedGameServerService = trustedGameServerService;
+		this.ratingsService = ratingsService;
+	}
+
+	public async Task RemoveNonActivatedAccountsAsync()
+	{
+		var nonActivatedAccountIds = await accountService.GetNonActivatedAccountsAsync();
+		if (!nonActivatedAccountIds.Any())
+		{
+			return;
+		}
```

Comment:
why do this check? if empty array list is returned, then none will be removed by `RemoveAccountAndAssociatedDataAsync`.